contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -485,4 +485,21 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
             uint256(vaultState.lockedAmount).sub(unlockedAssetAmount)
         );
     }
+
+    /**
+     * @notice Recovery function that returns an ERC20 token to the recipient
+     */
+    function recoverTokens(address token, address recipient)
```

Comment:
Should we add quantity as a param here? In case there are 2 people depositing the wrong assets. 

Otherwise, we can also recover to our account and re-distribute to the users.